aws/data_source_aws_iam_policy_document.go

```diff
@@ -84,6 +84,15 @@ func dataSourceAwsIamPolicyDocument() *schema.Resource {
 					},
 				},
 			},
+			"version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "2012-10-17",
+				ValidateFunc: validation.StringInSlice([]string{
+					"2008-10-17",
```

Comment:
If this version is selected then AWS won't accept interpolated variables, so perhaps we should then produce an error if the user attempts to use our custom `&{...}` variable syntax with that version number selected... otherwise we'll replace the `&{...}` with `${...}` but then that new form will just be interpreted literally.

Reply:
Would still want to keep the lack of a version to provide the 'latest' for people that want to ensure they are already on the latest version, just looking to ensure that if people want to either lock-in or need to utilize an older version are able to do so.  Thanks!